### PR TITLE
[homework #5] - CSS Advanced by Candelaria Barcelo

### PR DESCRIPTION
**Pull request by candebarcelo**
Hello @VitalikMelnyk , please review homework #5. Thank you!

EDIT: I saw on my previous homework's pull request that you said to make the branch from the main branch. I tried rebasing this branch since I had already turned it in, is this the correct way to rebase the branch?

### CODE REVIEW

File: homeworks/candelaria-barcelo/05-css-advanced/index.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="./favicon.ico" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
```
Comment (VitalikMelnyk) [inline homeworks/candelaria-barcelo/05-css-advanced/index.html:9]: Firstly, you need to create common styles for all screens.
For example, in each CSS file, you copy-paste the same CSS styles.
Please, create one common CSS file and remove unnecessary repeatable code.

File: homeworks/candelaria-barcelo/05-css-advanced/styles/mobile.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
+/* GLOBAL STYLES */
+
+body {
```
Comment (VitalikMelnyk) [inline homeworks/candelaria-barcelo/05-css-advanced/styles/mobile.css:3]: Please, create one common CSS file and remove unnecessary repeatable code.
And leave only styles for mobile.

File: homeworks/candelaria-barcelo/05-css-advanced/styles/tablet.css [hunk 3]
```diff
@@ -0,0 +1,222 @@
+/* GLOBAL STYLES */
+
+body {
```
Comment (VitalikMelnyk) [inline homeworks/candelaria-barcelo/05-css-advanced/styles/tablet.css:3]: Please, create one common CSS file and remove unnecessary repeatable code.
And leave only styles for tablet.

**Comment by candebarcelo** [comment]
Ok, thanks Vitalii. I will make a new pull request to delete all the other people's files that I accidentally commited here.

**Pull request closed without merging**